packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -153,8 +153,8 @@ function _domBreadcrumb(dom: BreadcrumbsOptions['dom']): (handlerData: HandlerDa
     try {
       const event = handlerData.event as Event | Node;
       target = _isEvent(event)
-        ? htmlTreeAsString(event.target, { keyAttrs, maxStringLength })
-        : htmlTreeAsString(event, { keyAttrs, maxStringLength });
+        ? getElementIdentifier(event.target, { keyAttrs, maxStringLength })
+        : getElementIdentifier(event, { keyAttrs, maxStringLength });
```

Comment:
Right now, `getElementIdentifier` isn't doing anything with the options passed to it, except passing it to `htmlTreeAsString` if there is no `data-component` annotation available. Would we want to make use of any of those options, like `maxStringLength` on the annotated component names as well?